controllers/controller_common.go

```diff
@@ -118,3 +120,9 @@ func applyClusterRefLabel(cluster *v1beta1.KafkaCluster, labels map[string]strin
 func SetNewKafkaFromCluster(f func(k8sclient client.Client, cluster *v1beta1.KafkaCluster) (kafkaclient.KafkaClient, func(), error)) {
 	newKafkaFromCluster = f
 }
+
+func ObjectManagedByClusterRegistry(object v1.Object) bool {
```

Comment:
```suggestion
func ObjectManagedByClusterRegistry(object metav1.Object) bool {
```

Reply:
It is already fixed by not pushed yet, as this PR is not ready for review.